### PR TITLE
Fixes 7

### LINKED ISSUE #7: upgrade to latest waggledance
In the current version I'm seeing that if one of the metastores is down, the whole thing does not work. In the latest version I believe there is better error handling.

### PR DESCRIPTION
**Pull request by mroark1m**
Fixes #7 
Updating waggledance to version 2.4.2. I also thought we could mark this as v1.0.0 to start using versions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,3 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - 2018-08-28
```
Comment (massdosage) [inline CHANGELOG.md]: Did you tag it already?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,3 +3,9 @@ All notable changes to this project will be documented in this file.
...
+### Changed
+- Changed waggledance version from 2.3.1 to 2.4.2. See [#7](https://github.com/ExpediaInc/apiary-waggledance-docker/issues/7).
+
+[Unreleased]: https://github.com/ExpediaInc/apiary-waggledance-docker/compare/v1.0.0...HEAD
```
Comment (massdosage) [inline CHANGELOG.md]: We normally don't bother with the git commit tags as it's just one more thing to maintain but perhaps it makes more sense for these kinds of projects that don't produce separate artifacts?

File: CHANGELOG.md (hunk 1 above)
Comment (mroark1m) [reply CHANGELOG.md]: No I assumed I would tag it once it hit master.

File: CHANGELOG.md (hunk 2 above)
Comment (mroark1m) [reply CHANGELOG.md]: @massdosage I guess I'm confused. This project produces a docker image, won't we tag the docker image?

**Pull request merged**